chalicelib_smaht/checks/wfr_checks.py

```diff
@@ -87,8 +90,8 @@ def search_query(self, query):
         self.add_items(search_response)
 
 
-@check_function(file_type="File", start_date=None, action="md5runsmaht_start")
-def md5runsmaht_status(connection, file_type="", start_date=None, **kwargs):
+@check_function(file_type="File", start_date=None, max_files=50, action="md5run_start")
```

Comment:
Consider restricting to only ReferenceFile and SubmittedFile types to prevent output files from overloading this space. We discussed not running md5 checksums on large intermediate files (OutputFile types) to save time and compute, and we don't want them to end up caught in here.

Reply:
Then again, maybe they wouldn't end up here because of the status info in the search.